### PR TITLE
Rename and refactor tests to make them clearer

### PR DESCRIPTION
**Pull request by julia-zack**

### CODE REVIEW

**Review by marcos-iov — APPROVED** [review]

File: rskj-core/src/test/java/co/rsk/peg/NonStandardErpFederationsTest.java [hunk 1]
```diff
@@ -337,48 +326,107 @@ void createLegacyErpRedeemScript_fromNonStandardErpBuilders() {
         );
         defaultThreshold = defaultKeys.size() / 2 + 1;
 
+        BtcECKey emergency0PublicKey = BtcECKey.fromPublicOnly(Hex.decode("0216c23b2ea8e4f11c3f9e22711addb1d16a93964796913830856b568cc3ea21d3"));
+        BtcECKey emergency1PublicKey = BtcECKey.fromPublicOnly(Hex.decode("0275562901dd8faae20de0a4166362a4f82188db77dbed4ca887422ea1ec185f14"));
+        BtcECKey emergency2PublicKey = BtcECKey.fromPublicOnly(Hex.decode("034db69f2112f4fb1bb6141bf6e2bd6631f0484d0bd95b16767902c9fe219d4a6f"));
+        emergencyKeys = Arrays.asList(
+            emergency0PublicKey, emergency1PublicKey, emergency2PublicKey
+        );
+        emergencyThreshold = emergencyKeys.size() / 2 + 1;
```
Comment (nathanieliov) [inline rskj-core/src/test/java/co/rsk/peg/NonStandardErpFederationsTest.java:335]: What about tests using different `standardThreshold` and `emergencyThreshold ` than n+1, for ex: negative value, zero, greater than M, etc?

File: rskj-core/src/test/java/co/rsk/peg/NonStandardErpFederationsTest.java [hunk 2]
```diff
@@ -337,48 +326,107 @@ void createLegacyErpRedeemScript_fromNonStandardErpBuilders() {
...
 
-        federator4PublicKey = BtcECKey.fromPublicOnly(Hex.decode("03fb8e1d5d0392d35ca8c3656acb6193dbf392b3e89b9b7b86693f5c80f7ce8581"));
+        // these values belong to the non-standard fed
+        BtcECKey federator0PublicKey = BtcECKey.fromPublicOnly(Hex.decode("0208f40073a9e43b3e9103acec79767a6de9b0409749884e989960fee578012fce"));
+        BtcECKey federator1PublicKey = BtcECKey.fromPublicOnly(Hex.decode("0225e892391625854128c5c4ea4340de0c2a70570f33db53426fc9c746597a03f4"));
+        BtcECKey federator2PublicKey = BtcECKey.fromPublicOnly(Hex.decode("025a2f522aea776fab5241ad72f7f05918e8606676461cb6ce38265a52d4ca9ed6"));
+        BtcECKey federator3PublicKey = BtcECKey.fromPublicOnly(Hex.decode("02afc230c2d355b1a577682b07bc2646041b5d0177af0f98395a46018da699b6da"));
+        BtcECKey federator4PublicKey = BtcECKey.fromPublicOnly(Hex.decode("03fb8e1d5d0392d35ca8c3656acb6193dbf392b3e89b9b7b86693f5c80f7ce8581"));
```
Comment (nathanieliov) [inline rskj-core/src/test/java/co/rsk/peg/NonStandardErpFederationsTest.java:405]: This same public key is being created and used in different tests. What do you think about moving it outside the method and converting it into a class instance variable?

**Review by nathanieliov — APPROVED** [review]

File: rskj-core/src/test/java/co/rsk/peg/NonStandardErpFederationsTest.java (hunk 1 above)
Comment (julia-zack) [reply rskj-core/src/test/java/co/rsk/peg/NonStandardErpFederationsTest.java:335]: Yes, good idea. I will create another pr for that

File: rskj-core/src/test/java/co/rsk/peg/NonStandardErpFederationsTest.java (hunk 2 above)
Comment (julia-zack) [reply rskj-core/src/test/java/co/rsk/peg/NonStandardErpFederationsTest.java:405]: Yes, good idea. I will create another pr for that

**Pull request merged**